packages/venia-concept/src/actions/checkout/__tests__/asyncActions.spec.js

```diff
@@ -169,21 +169,29 @@ describe('submitAddress', () => {
             cart: {},
             directory: { countries }
         }));
-
         await expect(submitAddress(payload)(...thunkArgs)).rejects.toThrow(
             'guestCartId'
         );
     });
+    test('submitAddress thunk dispatches action on incorrect state(region code)', async () => {
+        const invalidState = 'any_text';
+        const incorrectAddressMessage = `State "${invalidState}" is not an valid state abbreviation.`;
```

Comment:
Can we re-use the error message from `constants.js`? This way if it changes this test will change with it.

Reply:
Oops, nope, those aren't the same things.